config/routes.rb

```diff
@@ -1,5 +1,8 @@
 Rails.application.routes.draw do
   get '/restaurant', controller: :restaurant, action: :index
+
+  resources :dishes
```

Comment:
This creates all CRUD routes. Do you use them? I would suggest that you only create the ones you actually use.